packages/shared/providers.yaml

```diff
@@ -1424,6 +1424,22 @@ egnyte:
             description: The subdomain of your Egnyte account
             pattern: '^[a-z0-9_-]+$'
 
+elevenlabs:
+    display_name: Eleven Labs
+    categories:
+        - dev-tools
+    auth_mode: API_KEY
+    proxy:
```

Comment:
Can you add a `verification` prop. It should a simple call to an endpoint accessible to anyone with this api key